roles/bootstrap/README.md

```diff
@@ -24,3 +24,9 @@ Your playbook could look like this:
   roles:
     - role: ethpandaops.general.bootstrap
 ```
+
+### Enable reboot if required
+If you would like to enable the machines to reboot during bootstrap, then it can be set as an option.
+```sh
+ansible-playbook -i inventory.yaml -e bootstrap_reboot_if_required=true
```

Comment:
This is an invalid command, it's missing a playbook file .e.g `ansible-playbook playbook.yaml`.